### PR TITLE
meshery bash script fixed

### LINKED ISSUE #621: Running Meshery using bash command not showing error properly.
**Description**
No error message if "running Meshery using bash command"  produces a Permission denied error.
**Current Behavior**
On using  ` curl -L https://git.io/meshery | bash - ` command, if any error like "Permission denied" the command is not showing it up on the terminal after the first time you run the command.
**Expected Behavior**
If any error occurs while running the Meshery, errors should be shown on the terminal even after running the command multiple times.
**Screenshots**
bash_cmd_err

### PR DESCRIPTION
**Pull request by anandrkskd**
fixes #621 
fixed the error handling in meshery bash script. and removed permission issues

### CODE REVIEW

**Comment by leecalcote** [comment]
@nupur-thakur will you give this new script a try and see if it addresses the issue you've seen?

File: meshery [hunk 1]
```diff
@@ -61,7 +61,32 @@ printf "\nExtracting %s...\n" "$NAME"
 unzip ${PWD}/meshery.zip 
 
 printf "\nInstalling mesheryctl in /usr/local/bin.\n"
-mv ${PWD}/mesheryctl /usr/local/bin/mesheryctl
+if mv ${PWD}/mesheryctl /usr/local/bin/mesheryctl ; then
+  echo "mesheryctl installed"
```
Comment (leecalcote) [inline meshery]: @anandrkskd we should update their version of mesheryctl by overwriting their existing binary rather than aborting.

Comment (leecalcote) [reply meshery]: Given that this PR has merged, let's open a new issue to track the need to change this behavior.

File: meshery [hunk 2]
```diff
@@ -61,7 +61,32 @@ printf "\nExtracting %s...\n" "$NAME"
...
+else
+	echo "Cannot install mesheryctl. Aborting."
+  exit 1
+fi
+
+#Transfering permissions and ownership to client USER
+if [ "$SUDO_USER" == "" ] ;
+then
+	if 	chown $WHOAMI:$WHOAMI /usr/local/bin/mesheryctl ; then
+    echo "permissions moved to "$WHOAMI
```
Comment (leecalcote) [inline meshery]: This is a bit verbose. Users don't need to see this level of info.

File: meshery [hunk 3]
```diff
@@ -61,7 +61,32 @@ printf "\nExtracting %s...\n" "$NAME"
...
+  else
+  	echo "Cannot move permission. Aborting."
```
Comment (leecalcote) [inline meshery]: Good. Can we change to "Unable to write to `/usr/local/bin/mesheryctl`. Please verify write permission for $WHOAMI."?

File: meshery [hunk 4]
```diff
@@ -61,7 +61,32 @@ printf "\nExtracting %s...\n" "$NAME"
...
+    exit 1
+  fi
+else
+	if chown $SUDO_USER:$SUDO_USER /usr/local/bin/mesheryctl ; then
+    echo "permissions moved to "$SUDO_USER
+  else
+  	echo "Cannot move permission. Aborting."
```
Comment (leecalcote) [inline meshery]: Good. Can we change to "Unable to write to `/usr/local/bin/mesheryctl`. Please verify write permission for $SUDO_USER."?

File: meshery [hunk 5]
```diff
@@ -61,7 +61,32 @@ printf "\nExtracting %s...\n" "$NAME"
...
+then
+	if 	chown $WHOAMI:$WHOAMI /usr/local/bin/mesheryctl ; then
+    echo "permissions moved to "$WHOAMI
+  else
+  	echo "Cannot move permission. Aborting."
+    exit 1
+  fi
+else
+	if chown $SUDO_USER:$SUDO_USER /usr/local/bin/mesheryctl ; then
+    echo "permissions moved to "$SUDO_USER
```
Comment (leecalcote) [inline meshery]: We can remove or move this output behind a "--verbose" flag.

**Comment by anandrkskd** [comment]
Understood

**Pull request merged**